README.md

```diff
@@ -1,117 +1,5 @@
-# Notation Github Actions
-Github Actions for [Notation](https://notaryproject.dev/).
+# Notation Github Actions e2e-test-plugin
+This branch provides a mock plugin for notation Github actions E2E test workflows.
```

Comment:
We should set up a branch policy for the `e2e-test-plugin` branch.

Reply:
Sounds good. I think we could set it up after this PR gets merged?